--- i686-w64-mingw32.shared-cmake	2020-08-06 16:05:49.000000000 +0100
+++ x	2020-08-06 16:06:33.000000000 +0100
@@ -1,25 +1,25 @@
 #!/usr/bin/env bash
-echo "== Using MXE wrapper: /usr/lib/mxe/usr/bin/i686-w64-mingw32.shared-cmake"
+echo "== Using MXE wrapper: /usr/lib/mxe/usr/bin/i686-w64-mingw32.shared-cmake" >&2
 
 # https://cmake.org/cmake/help/latest/manual/cmake-policies.7.html
 # https://cmake.org/cmake/help/latest/variable/CMAKE_POLICY_DEFAULT_CMPNNNN.html
 POLICIES=(0017,0020)
 
 unset NO_MXE_TOOLCHAIN
-if echo -- "$@" | grep -Ewq "(--build|-E|--system-information)" ; then
+if echo -- "$@" | grep -Ewq "(--build|-E|--system-information|--version)" ; then
     NO_MXE_TOOLCHAIN=1
 fi
 if [[ "$NO_MXE_TOOLCHAIN" == "1" ]]; then
-    echo "== Skip using MXE toolchain: /usr/lib/mxe/usr/i686-w64-mingw32.shared/share/cmake/mxe-conf.cmake"
+    echo "== Skip using MXE toolchain: /usr/lib/mxe/usr/i686-w64-mingw32.shared/share/cmake/mxe-conf.cmake" >&2
     # see https://github.com/mxe/mxe/issues/932
     exec "/usr/lib/mxe/usr/x86_64-pc-linux-gnu/bin/cmake" "$@"
 else
     echo "     - cmake version 3.10.0"
     echo "     - warnings for unused CMAKE_POLICY_DEFAULT variables can be ignored"
-    echo "== Using MXE toolchain: /usr/lib/mxe/usr/i686-w64-mingw32.shared/share/cmake/mxe-conf.cmake"
-    echo "== Using MXE runresult: /usr/lib/mxe/usr/share/cmake/modules/TryRunResults.cmake"
+    echo "== Using MXE toolchain: /usr/lib/mxe/usr/i686-w64-mingw32.shared/share/cmake/mxe-conf.cmake" >&2
+    echo "== Using MXE runresult: /usr/lib/mxe/usr/share/cmake/modules/TryRunResults.cmake" >&2
     if ! ( echo "$@" | grep --silent "DCMAKE_BUILD_TYPE" ) ; then
-        echo '== Adding "-DCMAKE_BUILD_TYPE=Release"'
+        echo '== Adding "-DCMAKE_BUILD_TYPE=Release"' >&2
         set -- "-DCMAKE_BUILD_TYPE=Release" "$@"
     fi
     exec "/usr/lib/mxe/usr/x86_64-pc-linux-gnu/bin/cmake" \
